### PR TITLE
Added Serbian cyrilic translation.

### PR DESCRIPTION
**Pull request by ntadic**
Added Serbian cyrilic translation.

## Description
Added translation file for Serbian cyrilic

## Motivation and Context
Supporting a new language.

## How Has This Been Tested?
It is not tested since it is a translation file.

## Markup:
<details>

```vue
// Paste your FULL Playground.vue here
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have created a PR in the [documentation](https://github.com/vuetifyjs/vuetifyjs.com) with the necessary changes.

### CODE REVIEW

**Comment by KaelWD** [comment]
The ISO 15924 language tag is `sr-Cyrl`

**Comment by jacekkarczmarczyk** [comment]
Can you please also add it to https://github.com/vuetifyjs/vue-cli-plugin-vuetify/blob/dev/prompts.js#L63?
Thank you in advance

**Comment by ntadic** [comment]
> Can you please also add it to https://github.com/vuetifyjs/vue-cli-plugin-vuetify/blob/dev/prompts.js#L63?
> Thank you in advance

Added. Cheers.

File: src/stylus/components/_toolbar.styl [hunk 1]
```diff
@@ -120,6 +120,9 @@ theme(v-toolbar, "v-toolbar")
 
   .v-input
     margin: 0
+  
+  .v-text-field
```
[the remark this answers is not in the document]
Comment (ntadic) [reply src/stylus/components/_toolbar.styl]: Removed

**Pull request merged**